cmd/nerdctl/start.go

```diff
@@ -119,6 +121,14 @@ func startContainer(ctx context.Context, container containerd.Container, flagA b
 		return err
 	}
 	flagT := process.Process.Terminal
+	var con console.Console
+	if flagA && flagT {
```

Comment:
cc @yzxiu 
```suggestion
	if flagT {
```

Reply:
if `process.Process.Terminal` is true, `nerdctl start demo` will be allocated a term (con)

Reply:
if `start` without `-a`, after nerdctl starts the container, it exits directly.
https://github.com/containerd/nerdctl/blob/40d341dac245a70a6ee4ac43567f2d2818fe3167/cmd/nerdctl/start.go#L160-L162
`con` doesn't do anything here

Reply:
@AkihiroSuda cc